Stratis.Bitcoin/Signals.cs

```diff
@@ -0,0 +1,43 @@
+﻿using NBitcoin;
+using Stratis.Bitcoin.Utilities;
+
+namespace Stratis.Bitcoin
+{
+    public interface ISignals
+    {
+        ISignaler<Block> Blocks { get; }
+        ISignaler<Transaction> Transactions { get; }
+
+        void Signal(Transaction trx);
```

Comment:
We can remove this methods as its more understandable to use the property name when broadcasting, but this is for later.